modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -5,41 +5,47 @@ endif::[]
 :imagesdir: ../../assets/images/
 
 == Example
+//  I SUGGEST OMITTING THIS SUBTITLE, because the topic set already has a title (Amazon SQS Examples - Mule 4) and because this subtitle is at the same level as the two subtopics that it contains. Also, the title "Example" is not useful, but the two subtitles "Create a Flow..." are. //
 
```

Comment:
I SUGGEST OMITTING THIS SUBTITLE, because the topic set already has a title (Amazon SQS Examples - Mule 4) and because this subtitle is at the same level as the two subtopics that it contains. Also, the title "Example" is not useful, but the two subtitles "Create a Flow..." are.